### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by oldmcdonnell**
code of conduct

### CODE REVIEW

**Review by n3ndor — DISMISSED** [review]
Looks nice, I like the little green colors - looks really professional!

**Comment by Frankdias92** [comment]
we can merger this for ours main branch right? this merge will merge 49 new files to main just for the code of CODE_OF_CONDUCT. we are working do branch `dev`

I suggest to to make a new pr for our branch dev.

let me know if you agree with this [oldmcdonnell](https://github.com/oldmcdonnell)

**Comment by oldmcdonnell** [comment]
updated to dev

**Review by davideastmond — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<<<<<<< HEAD
```
Comment (davideastmond) [inline .gitignore:1]: It looks like merge syntax got here. If we remove these manually, just double check that the .gitignore looks good and we can merge this 👍🏿

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+node_modules
+=======
```
Comment (davideastmond) [inline .gitignore:3]: Here too

File: .gitignore [hunk 3]
```diff
@@ -38,3 +41,4 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+>>>>>>> dev
```
Comment (davideastmond) [inline .gitignore:44]: Here too. Just needs to be deleted :)

**Review by davideastmond — COMMENTED** [review]
You should be able to delete the merge syntax and we can get this into dev

**Pull request closed without merging**